roles/common/templates/pulp-operator-sa.yaml.j2

```diff
@@ -1,5 +1,9 @@
 apiVersion: v1
-{% if image_pull_secrets is defined %}
+{% if image_pull_secret is defined %}
+# image_pull_secret is deprecated in favor of image_pull_secrets
+imagePullSecrets:
+  - name: {{ image_pull_secret }}
+{% elif image_pull_secrets | length > 0 %}
```

Comment:
Good idea on keeping the `image_pull_secret` for backward compatibility! :+1: 

The only thing that I think that we could modify is the ` {% elif image_pull_secrets | length > 0 %}`.
It would probably be better to keep the condition of `image_pull_secrets is defined` because, for situations in which someone configured the secrets but later wants to remove them, if we keep the `length > 0` the operator will not update the `serviceaccount`, keeping the old secrets on the `serviceaccount` (not reflecting the configuration from `pulp CR instance`).